18F/identity-idp
# LG-6066: Implement clipboard button on FSMv2 Personal Key step

**Why**: So that we can retain feature parity with the existing screen.

**Testing Instructions:**

1. Set `idv_api_enabled: "true"` in local `config/application.yml`
2. Go to: http://localhost:3000/verify/v2/personal_key
3. Click "Copy"
4. Paste text value somewhere (e.g. browser address bar)
5. Note that pasted value is "0000-0000-0000-0000"

![image](https://user-images.githubusercontent.com/1779930/163400434-e3ff8dcb-5551-40fe-8bdb-398bcf192c30.png)
